test/data/testinit.js

```diff
@@ -160,7 +165,9 @@ this.ajaxTest = function( title, expect, options ) {
 					if ( options.teardown ) {
 						options.teardown();
 					}
-					start();
+
+					// Make sure all events will be called before done()
+					setTimeout( done );
```

Comment:
This seems dangerous; could you explain why a single `setTimeout` is guaranteed to fix it?